lib/shopify-cli/tasks/js_deps.rb

```diff
@@ -15,7 +15,10 @@ def call(ctx)
         @dir = ctx.root
 
         CLI::UI::Frame.open("Installing dependencies with #{installer}...") do
-          send(installer)
+          CLI::UI::Spinner.spin("Installing dependencies...") do |spinner|
+            send(installer)
+            spinner.update_title("#{@dep_number} #{installer} dependencies installed")
```

Comment:
This `@dep_number` thing is specific to `npm` which means this will be `nil` if we ever have a yarn-based project (currently unused). I think we should just move this spinner into the `npm` method for now.

Reply:
I was wondering that very thing and also was like 'we are not using yarn why is it here' I will move